modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/src/main/webapp/WEB-INF/views/view.xhtml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<f:view
+	xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+>
+	<h:head>
+		<h:outputStylesheet library="css" name="main.css" />
+	</h:head>
+	<h:body>
+		<h:panelGroup id="panelId" layout="block">
+			<h:outputText styleClass="bold-hello-world" value="#{i18n['sample-bean-portlet']}" />
+			<p><h:outputText styleClass="bold-hello-world" value="#{myBacking.greeting}" /></p>
+		</h:panelGroup>
+		<ul>
```

Comment:
@Kvale1733, because your testcase only test the portlet is still accessible. Not check the portlet display content. Please remove the [code1](https://github.com/liferay-core-infra/liferay-portal/blob/8842f850161fe5e9746ab97a7213f60d2e89dda5/modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/src/main/webapp/WEB-INF/views/view.xhtml#L8-L10) and [code2](https://github.com/liferay-core-infra/liferay-portal/blob/8842f850161fe5e9746ab97a7213f60d2e89dda5/modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/src/main/webapp/WEB-INF/views/view.xhtml#L12-L15). After this, **main.css**, **MyBacking.java** and **Greeting.java** are unnecessary. Please also removed them.